src/protocol/protocol.ml

```diff
@@ -22,6 +22,9 @@ let apply_core_tezos_operation state tezos_operation =
   in
   let core_state =
     Core.State.apply_tezos_operation state.core_state tezos_operation in
+  Printf.eprintf "Received Tezos operation, core state is now:\n%!";
+  Printf.eprintf "%s\n%!"
+    (Core.State.to_yojson state.core_state |> Yojson.Safe.to_string);
```

Comment:
remove me

Reply:
Could use `Logs.debug` to have an easy way of re-adding them back if it's needed